docs/functions/json.md

```diff
@@ -0,0 +1,93 @@
+Following functions help you handle JSON documents.
+
+!!! important "The document object given to those functions could be of any type as it is always serialize as JSON string first"
+
+
+# JsonPath
+
+!!! note "Object jsonPath(Object document, String jsonPath)"
+
+    Read the given JSON path from the given document.
+
+    **Parameters** :
+
+    * `document` : The context document in which the path will be used.
```

Comment:
We may emphasize "context" means scenario execution context, or during execution

Reply:
=> json